### PR TITLE
Release arm64 binary for m1 macs

### PR DESCRIPTION
**Pull request by chrisdostert**
This PR updates release to include an arm64 build for our friends using m1 macs.

### CODE REVIEW

**Review by gary-affonso-remitly — APPROVED** [review]
I am completely unqualified to review this.  Nor am I under the impression my approval is of any value, I don’t think I’m a “maintainer” thus my approval won’t unblock the build. 

That said, I’m eager for this change.  Consider my approval a shout of encouragement.

**Review by gary-affonso-remitly — APPROVED** [review]

**Review by zachpuck — APPROVED** [review]

**Comment by chrisdostert** [comment]
Note) Since what's in main branch is vnext and we don't want to wait on a release cycle just to have an m1 binary, I've generated and uploaded an arm64 artifact for the [current (0.1.48) release](https://github.com/opctl/opctl/releases/tag/v0.1.48)

**Pull request merged**
